### PR TITLE
feat: bid auction

### PR DESCRIPTION
**Pull request by topether21**
Instructions here: https://github.com/deezy-inc/deezy-place/pull/298

### CODE REVIEW

**Review by habibitcoin — CHANGES_REQUESTED** [review]
Pretty much good to go, but would want to understand the fee behavior a bit better. As mentioned on the other PR, we seem to be underestimating for some reason.

The issue is going to either be:
1. Some problem with estimating size in src/app/crypto.ts:29, but this looks to be correct
2. Some problem with the feeRate on src/app/crypto.ts:30. It looks like this can end up as either the value selected by the bidder, a default rate, or a recommended rate. Maybe we're accidentally not invoking the value selected by the bidder? Is that intentional perhaps?

**Comment by topether21** [comment]
@habibitcoin it is fixed, take a look at 

image

**Comment by topether21** [comment]
I added a console.log and before merge I can delete it.

image

File: src/app/psbt.ts [hunk 1]
```diff
@@ -218,17 +218,23 @@ const Psbt = function (config) {
...
             const hex = tx.toBuffer().toString('hex');
             const fullTx = bitcoin.Transaction.fromHex(hex);
-            await axios.post(`https://mempool.space/api/tx`, hex);
-
+            // It is added in order to debug the transaction
+            if (!skipBroadcastTx) {
+                // This console.log is just for testing purposes
+                // We can remove it later
+                // TODO: remove it
+                console.log('[Broadcasting tx]', hex);
```
Comment (habibitcoin) [inline src/app/psbt.ts]: yeah lets remove, but good job finding the smoking gun!

**Review by habibitcoin — APPROVED** [review]
Looks good! lets just remove the log statements prior ot merge

**Review by andredevjs — APPROVED** [review]

**Pull request merged**